support/camel-k-maven-plugin/src/main/java/org/apache/camel/k/tooling/maven/GenerateCatalogMojo.java

```diff
@@ -240,6 +240,19 @@ public void execute() throws MojoExecutionException, MojoFailureException {
                         .build()
                 );
             }
+            if (capabilitiesExclusionList != null && !capabilitiesExclusionList.contains("resume")) {
+                runtimeSpec.putCapability(
+                        "resume",
```

Comment:
Should we add some additional metadata here ?  
Like, how can I pick a different resume strategy ?
I'd like to avoid having to introduce braking change later

Reply:
Maybe we could specialize this as well? I.e.: instead of `resume`, name it as `resume-kafka`. Then we can reserve the capability named `resume` to later if we need to. 

wdyt?

Reply:
That would be ok

Reply:
I've just renamed it to `resume-kafka`. Please let me know if you think we should stick to something else.